userspace/libsinsp/user.cpp

```diff
@@ -243,59 +357,110 @@ scap_userinfo *sinsp_usergroup_manager::add_user(const string &container_id, uin
 	scap_userinfo *usr = get_user(container_id, uid);
 	if (!usr)
 	{
-#ifdef HAVE_PWD_H
-		struct passwd *p = nullptr;
-		if (container_id.empty() && !name)
+		bool do_notify{false};
```

Comment:
I'd renamed this bool to `usermap_updated` or something more meaningful!

Reply:
`inserted`